{"name": "Авторизация курьера — проверка ошибок", "status": "passed", "description": "Проверка авторизации негативными сценариями", "steps": [{"name": "Сценарий: Неправильный логин", "status": "passed", "start": 1757417941919, "stop": 1757417942545}, {"name": "Проверить код ответа", "status": "passed", "start": 1757417942545, "stop": 1757417942545}, {"name": "Проверить сообщение об ошибке", "status": "passed", "start": 1757417942545, "stop": 1757417942545}], "parameters": [{"name": "description", "value": "'Неправильный логин'"}, {"name": "creds", "value": "{'login': 'wrong_login', 'password': 'pass456'}"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1757417941919, "stop": 1757417942545, "uuid": "97ea8ed4-95b4-4a43-9988-4a4d57aeda8c", "historyId": "797a7c58ccbf0176836696865db0c3a2", "testCaseId": "19c43b3cacae3758579e377968a7e303", "fullName": "tests.test_login_courier.TestCourierLogin#test_login_errors", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Ksu"}, {"name": "thread", "value": "16324-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestCourierLogin"]}